1. 
        public class Exception3
        {
            static public void main(String [] arg)
            {
                try
                {
                    int n;
                    int[] arr =new int[n];
                }
                catch(Exception e)
                {
                    System.out.println(e);
                }
            }
        }
2.
        public class Exception3
        {
            static public void main(String [] arg)
            {
                try
                {
                    int[] arr =new int[n];
                }
                catch(Exception e)
                {
                    System.out.println(e);
                }
            }
        }